src/Indicator.vala

```diff
@@ -210,6 +211,37 @@ public class Notifications.Indicator : Wingpanel.Indicator {
             warning ("Failed to open notifications settings: %s", e.message);
         }
     }
+
+    private void update_tooltip () {
+        uint number_of_notifications = Session.get_instance ().get_session_notifications ().length ();
+        int number_of_apps = nlist.app_entries.size;
+
+        string description;
+        string accel_label;
+
+        if (notify_settings.get_boolean ("do-not-disturb")) {
+            accel_label = _("Middle-click to disable Do Not Disturb");
+        } else {
+            accel_label = _("Middle-click to enable Do Not Disturb");
+        }
+
+        accel_label = """<span weight="600" size="smaller" alpha="75%">%s</span>""".printf (accel_label);
+
+        switch (number_of_notifications) {
+            case 0:
+                description = _("No notifications");
+                break;
+            case 1:
+                description = _("1 notification");
+                break;
+            default:
+                /* Anything else */
+                description = _("%u notifications from %i %s".printf (number_of_notifications, number_of_apps, ngettext ("app", "apps", number_of_apps)));
```

Comment:
This should also use nggettext for the word "notifications". Maybe @ryonakano has advice on the best way to structure this for translators since the phrase contains two plurals

Reply:
Hmmm. I'm currently using switch case statements to choose the tooltip text between

- No notifications
- 1 notification
- `$number_of_notifications` notifications from `$number_of_apps` app/apps.

I don't think using nggettext for notifications is needed since we'll only jump into the third case when there is more than one notification whereas nggettext was used for app/apps since "2 notifications from 1 app" is a possibility as pointed out by @cassidyjames.

Please let me know if I'm missing something.

Reply:
Yeah, I think we also need to use ngettext for languages that have multiple plural forms like Czech. Also, we may want to left a comment for translators, like this:

```suggestion
                /// TRANSLATORS: A tooltip text for the indicator representing the number of notifications.
                /// e.g. "2 notifications from 1 app" or "5 notifications from 3  apps"
                description = _("%s from %s".printf (
                    ngettext ("%u notification", "%u notifications",  number_of_notifications).printf (number_of_notifications),
                    ngettext ("%i app", "%i apps", number_of_apps).printf (number_of_apps)
                ));
```

Reply:
Alright, thanks for pointing it out